protocol/warden_client/src/application.rs

```diff
@@ -6,8 +6,8 @@ pub struct ApplicationConfig {
     pub name: String,
     pub version: String,
     pub image_registry: String,
-    pub image_storage_size_mb: usize,
-    pub data_storage_size_mb: usize
+    pub image_storage_size_mb: u32,
+    pub data_storage_size_mb: u32
```

Comment:
Minor: Why u32?. usize looked perfect for such task.

Reply:
I thought that it is more precise because `usize` is very often use for indexing arrays or vectors and it's size is dependent on the platform.